### ISSUE TITLE
Dilated convolutions support

### ISSUE THREAD

**Issue by dado330** [google-research/receptive_field#7]
@andrefaraujo, does the library support dilations?

If not, would it be possible to add the option since they are a basic parameter, for example, in Conv2D layers?

The formula for computing the receptive field of a layer using dilated convolutions can be found here: https://theaisummer.com/receptive-field/

Anyway, thanks for your work

**Comment by andrefaraujo** [comment]
yes, it should be simple to add. Would you like to help with that? :)
